docs/platforms/javascript/common/apis.mdx

```diff
@@ -0,0 +1,941 @@
+  end the session first.
+</SdkApi>
+
+## User Feedback
+
+<SdkApi
+  name="captureFeedback"
+  signature="function captureFeedback(feedback: Feedback, hint?: Hint): string"
+  parameters={[
+    {
+      name: "feedback",
+      type: {
+        name: "Feedback",
+        properties: [
+          { name: "message", type: "string", required: true },
+          { name: "name", type: "string" },
+          { name: "email", type: "string" },
+          { name: "url", type: "string" },
+          { name: "source", type: "string" },
+          {
+            name: "associatedEventId",
+            type: "string",
+            description: "The event id that this feedback is associated with.",
+          },
+          { name: "tags", type: "Record<string, string>" },
+        ],
+      },
+      description: "The feedback to capture.",
+    },
+    {
+      name: "hint",
+      type: {
+        name: "Hint",
+        properties: [
+          {
+            name: "captureContext",
+            type: {
+              name: "CaptureContext",
+              properties: [
+                {
+                  name: "user",
+                  type: {
+                    name: "User",
+                    properties: [
+                      { name: "id", type: "string | number" },
+                      { name: "email", type: "string" },
+                      { name: "ip_address", type: "string" },
+                      { name: "username", type: "string" },
+                    ],
+                  },
+                },
+                {
+                  name: "level",
+                  type: '"fatal" | "error" | "warning" | "log" | "info" | "debug"',
+                },
+                {
+                  name: "extra",
+                  type: "Record<string, unknown>",
+                  description:
+                    "Additional data that should be sent with the exception.",
+                },
+                {
+                  name: "tags",
+                  type: "Record<string, string>",
+                  description:
+                    "Additional tags that should be sent with the exception.",
+                },
+                {
+                  name: "contexts",
+                  type: "Record<string, Record<string, unknown>>",
+                },
+                { name: "fingerprint", type: "string[]" },
+              ],
+            },
+            description:
+              "Optional additional data to attach to the Sentry event.",
+          },
+        ],
+      },
+      description:
+        "Optional hint object containing additional information about the feedback.",
+    },
+  ]}
+>
+  Send user feedback to Sentry.
+</SdkApi>
+
+<SdkApi
+  name="getFeedback"
+  signature="function getFeedback(): ReturnType<feedbackIntegration> | undefined"
+>
+  Get the feedback integration, if it has been added. This can be used to access
+  the feedback integration in a type-safe way.
+</SdkApi>
+
+<SdkApi
```

Comment:
Something's busted here. It doesn't render past sendFeedback.

Reply:
huh, pushed a fix hopefully!